mmdet/models/roi_heads/bbox_heads/bbox_head.py

```diff
@@ -275,35 +275,97 @@ def get_bboxes(self,
                    scale_factor,
                    rescale=False,
                    cfg=None):
+        """Transform network output for a batch into bbox predictions.
+
+        Args:
+            rois (Tensor): Boxes to be transformed. Has shape (N, 5)
+               or (B, N, 5)
+            cls_score (list[Tensor] or Tensor): Box scores for
+              each scale level, each is a 4D-tensor, the channel number is
+              num_points * num_classes.
+            bbox_pred (Tensor, optional): Box energies / deltas for each scale
+                level, each is a 4D-tensor, the channel number is
+                num_classes * 4.
+            img_shape (Sequence[int] or torch.Tensor or Sequence[
+                Sequence[int]], optional): Maximum bounds for boxes, specifies
+                (H, W, C) or (H, W). If rois shape is (B, N, 4), then
+                the max_shape should be a Sequence[Sequence[int]]
+                and the length of max_shape should also be B.
+            scale_factor (ndarray): Scale factor of the image arange as
+                (w_scale, h_scale, w_scale, h_scale).
+            rescale (bool): If True, return boxes in original image space.
+                Default: False.
+            cfg (obj:`ConfigDict`): `test_cfg` of Bbox Head. Default: None
+
+        Returns:
+            tuple[list[Tensor], list[Tensor]] or tuple[Tensor, Tensor]: The
+                first list includes batch tensor and the shape is (n, 5)
+                where 5 represent (tl_x, tl_y, br_x, br_y, score) and the
+                score between 0 and 1. The second list also includes tensors
+                of the same length as the first list, and the shape is (n,)
+                where each element represents the class label of the
+                corresponding box.
```

Comment:
The first list contains the boxes of all images in a  batch，each tensor has the shape  
 (num_bboexes, 5) and last dimension 5 represent (tl_x, tl_y, br_x, br_y, score). Each Tensor in the second list is the labels with shape (num_bboexes, ).